api/user/service/update.js

```diff
@@ -0,0 +1,17 @@
+import update from '../repository/update.js';
+import findById from '../repository/getById.js';
+
+async function updateUser(validatedUser) {
+  const user =validatedUser.value;
```

Comment:
you need to pass the user object from the controller.
Service should know nothing about validation that was performed in the controller